soc/openhwgroup/cva6/soc_poweroff_fpga.c

```diff
@@ -0,0 +1,33 @@
+/*
+ * Busy-wait-forever poweroff for the CVA6 CPU on the FPGA
+ *
+ * Copyright(c) 2024, CISPA Helmholtz Center for Information Security
+ * SPDX - License - Identifier : Apache-2.0
+ */
+#include "cva6.h"
+
+#include <stdint.h>
+#include <stdio.h>
+
+#include <zephyr/sys/poweroff.h>
+
+void z_cva6_finish_test(const int32_t status)
+{
+
+	printf("Finishing test with status %u-", status);
+
+	if (status == 0) {
+		printf("TEST SUCCESS!\n");
+	} else {
+		printf("TEST FAIL!\n");
+	}
+
+	sys_poweroff();
+}
+
+void z_sys_poweroff(void)
+{
+	printf("System poweroff!\n");
+	for (;;) {
+	}
```

Comment:
Doesn't seem to look like a poweroff, could be misleading for the user.

Reply:
I will remove this file from the PR. 
I have only added it such that my samples for the testbench board compile on the other CVA6 boards as well. I think a better solution is to simply depend on the "proper" power off at configuration time for the samples.